codegen/core/src/main/java/software/amazon/smithy/python/codegen/generators/SetupGenerator.java

```diff
@@ -274,6 +284,38 @@ private static void writeReadme(
         });
     }
 
+    private static void writeLicense(
+            GenerationContext context
+    ) {
+        context.writerDelegator().useFileWriter("LICENSE", writer -> {
```

Comment:
How is this code AWS specific.  I haven't tested it, so I could be wrong, but I think this will get generated for all SDKs, even non-AWS ones.  

Does Smithy force a license that a customer must use?  I would expect this to be a generic license that was smithy's default, then have that be overwritten by an AWS specific interceptor.

Reply:
This PR now only adds our LICENSE and NOTICE file to AWS clients.